frontend/src/components/Workspace/WorkspaceTabs.tsx

```diff
@@ -1,31 +1,58 @@
-import { FC, SyntheticEvent } from "react"
+import { FC, SyntheticEvent, useCallback } from "react"
 import { useDispatch, useSelector } from "react-redux"
 
 import Tab from "@mui/material/Tab"
 import Tabs from "@mui/material/Tabs"
 
-import { selectActiveTab } from "store/slice/Workspace/WorkspaceSelector"
+import Loading from "components/common/Loading"
+import { StatusROI } from "components/Workspace/Visualize/Plot/ImagePlot"
+import { cancelRoi } from "store/slice/DisplayData/DisplayDataActions"
+import { selectLoadingVisualize } from "store/slice/DisplayData/DisplayDataSelectors"
```

Comment:
unused, pls remove

Reply:
@ReiHashimoto i removed it, please check again

Reply:
@ReiHashimoto i removed it, please check again